### PR TITLE
Fix #143

### LINKED ISSUE #143: Gui not working on Linux Mint 17.3
I tried with sudo. Here the output 

```shell
Traceback (most recent call last):
  File "/opt/numix-folders/gui/gui", line 220, in on_activate
    self.colourentries[i].set_max_width_chars(7)
AttributeError: 'Entry' object has no attribute 'set_max_width_chars'
```

### PR DESCRIPTION
**Pull request by bil-elmoussaoui**
This will fix #143 
The gui with older versions of gtk
screenshot from 2016-01-06 12-44-44
With Gtk >= 3.12
screenshot from 2016-01-06 12-46-16

### CODE REVIEW

**Comment by dirtydancing** [comment]
Works fine just as before with Arch/GNOME, Gtk 3.18 :+1:

File: gui/gui [hunk 1]
```diff
@@ -210,20 +210,22 @@ class NumixFoldersGUI(Gtk.Application):
...
             self.colourentries.append(Gtk.Entry())
             self.colourentries[i].set_text(self.colour[i])
             self.colourentries[i].set_max_length(7)
-            self.colourlabels[i].set_text(colour_labels[i])
-            self.colourentries[i].set_max_width_chars(7)
             self.colourentries[i].set_width_chars(7)
+            if Gtk.get_major_version() >= 3 and Gtk.get_minor_version() >= 12:
+                self.colourentries[i].set_max_width_chars(7)
+                self.colourlabels.append(Gtk.Label())
+                self.colourlabels[i].set_text(colour_labels[i])
```
Comment (wa4557) [inline gui/gui]: Aren't line 222 and 223 something that is necessary for both cases? (not only if Gtk >=3.12)

**Comment by bil-elmoussaoui** [comment]
@wa4557 Having the label + the large entry will make the gui look ugly.. and we will need to change a lot in the code.

**Comment by Foggalong** [comment]
Anyone with GTK 3.10 verify this?

**Comment by bil-elmoussaoui** [comment]
@Foggalong https://github.com/numixproject/numix-folders/issues/143#issuecomment-169439643

**Comment by Foggalong** [comment]
:+1:

**Pull request merged**